src/celestia/qt/qtappwin.cpp

```diff
@@ -1228,25 +1273,20 @@ void CelestiaAppWindow::createMenus()
     textureResolutionMenu->addAction(actions->mediumResAction);
     textureResolutionMenu->addAction(actions->highResAction);
 
-#ifdef VIDEO_SYNC
-    displayMenu->addSeparator();
-    displayMenu->addAction(actions->toggleVSyncAction);
-#endif
-
-    QMenu *fpsMenu = new QMenu(_("FPS rate"), this);
+    QMenu *fpsMenu = new QMenu(_("&FPS control"), this);
 
-    std::array<int, 5> fps_array = { 0, 15, 30, 60, 120 };
-
-    for (auto ifps : fps_array) {
-        QAction *fps = new QAction(ifps == 0 ? _("Auto") : QString::number(ifps), this);
-        fps->setData(ifps);
-        connect(fps, &QAction::triggered, this, &CelestiaAppWindow::setFPS);
-        fpsMenu->addAction(fps);
+    fpsActions = new FPSActionGroup(this);
+    for (const auto a : fpsActions->actions())
```

Comment:
you have forgotten `&` after `auto`. you don't need a new intermediate object here.